hypervisor/arch/x86/trampoline.c

```diff
@@ -110,7 +110,7 @@ uint64_t prepare_trampoline(void)
 	uint64_t size, dest_pa;
 
 	size = (uint64_t)(&ld_trampoline_end - &ld_trampoline_start);
```

Comment:
Can you use same name and same method with above code?

Reply:
Even we can define an inline API somewhere in head file.

Reply:
OK, I will add an inline api.

Reply:
`size' is used to memory copy and cache flush,  should I round page up here.

Reply:
You can rename them to trampiline_size, trampoline_pa

Reply:
OK， I will rename `size` as `trampoline_size`, and when alloc memory it will round page up inside `e820_alloc_memory`.